components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/validators/TokenValidationHandler.java

```diff
@@ -408,9 +410,21 @@ private OAuth2IntrospectionResponseDTO validateAccessToken(OAuth2TokenValidation
             }
 
         } else {
-
             try {
                 accessTokenDO = OAuth2Util.findAccessToken(validationRequest.getAccessToken().getIdentifier(), false);
+                List<String> allowedScopes = OAuthServerConfiguration.getInstance().getAllowedScopes();
+                String[] requestedScopes = accessTokenDO.getScope();
+                List<String> scopesToBeValidated = new ArrayList<>();;
```

Comment:
additional `;`

Reply:
removed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1468/commits/a1fc7ccf713f226608dc667d1bdcf13cd22ad83d